front/lib/api/assistant/conversation.ts

```diff
@@ -330,7 +332,6 @@ export type UserMessageNewEvent = {
 export type UserMessageErrorEvent = {
   type: "user_message_error";
   created: number;
-  messageId: string;
```

Comment:
we could keep  this field no? better to give a messageId if we can (undefined if we can't), + consistency with AgentMessageErrorEvent

Reply:
Well I would make the argument that this is inconsistent in the case of editMessage to send the id of the old message since this event is about anything but that old message